### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by presidentelect**
minor typo fixes

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Foam is a like a bathtub: _What you get out of it depends on what you put into i
 
 Like the soapy suds it's named after, **Foam** is mostly air.
 
-1. The editing experience of **Foam** is powered by VS Code, enhanced by workspace settings that glue together [[recommended-extensions]] and preferences optimised for writing and navigating information.
+1. The editing experience of **Foam** is powered by VS Code, enhanced by workspace settings that glue it together [[recommended-extensions]] and preferences optimised for writing and navigating information.
```
Comment (jevakallio) [inline docs/index.md:50]: In this change the original is as intended. Workspace settings glue recommended extensions and preferences together, but with inverted word order "glue together". Not sure if this is grammatically correct, but the suggested change is definitely not.

Roll this one back?

File: docs/index.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ Head over to [[recipes]] for some useful patterns and ideas!
 
 The goal of **Foam** is to be your personal companion on your quest for knowledge.
 
-It's is currently about "10% ready" relative to all the features I've thought of, but I've only thought of ~1% of the features it could have, and I'm excited to learn from others.
+It's currently about "10% ready" relative to all the features I've thought of, but I've only thought of ~1% of the features it could have, and I'm excited to learn from others.
```
Comment (jevakallio) [inline docs/index.md:90]: Good catch!

File: docs/index.md (hunk 1 above)
Comment (presidentelect) [reply docs/index.md:50]: ok, I see where I misread,  the linked doc caused me to read it like a pause/comma was there.

Comment (presidentelect) [reply docs/index.md]: I have reverted this change.

**Pull request merged**